trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
-work and they laud excellent and especially first contributions. The Trusted Committer's job
-in all this is primarily to set an example for others, similarly to setting an
-example for the level of software quality that is expected.  If necessary, the
-Trusted Committers are the ones who should create and enact a code of conduct for the
-community. Should there be community members whose behavior is detrimental or
-even toxic to the community health, it is the Trusted Committer's responsibility to either
-try and change or contain this or, in the worst case, to remove people from the
-community. Trusted Committers should create opportunities for people to get together
-regularly and get to know each other personally.
+Another, more obvious reason for people to stick around is that they
+enjoy working with other members of the community, including the Trusted Committers. A thriving community is one where members treat
+and communicate with each other with the utmost respect. Contributions are
+treated like gifts or donations, and excellent (especially
+first) contributions are lauded. The Trusted Committer’s job in all this is primarily to set an
+example for others, similar to setting an example for the level of
+software quality expected. If necessary, the Trusted Committers are the ones
+who should create and enact a code of conduct for the community. If
+there are community members whose behavior is detrimental or toxic to
+the community’s health, it is the Trusted Committer’s responsibility to address this. 
+Trusted Committers should create opportunities for people to get together
+regularly and get to know each other personally and to peacefully resolve conflicts as they arise.
 
-Another reason for people to stick around is that their
-work in an InnerSource community was an excellent opportunity to acquire new
-skills and to grow personally. This is again where the role of the Trusted Committer is really
-important. Trusted Committers often become mentors for junior developers, and they explicitly
-spend time during pull requests not only to point out areas for improvement but
-also explain in detail why something needs to be improved.  This explanation includes the theory
-or experience behind it and offering suggestions on how it is best done. By doing
-so, the Trusted Committers will often be able to increase the speed of learning in their
-communities far beyond what is common in traditional software development
-projects.
+People also tend to stick around because working in an
+InnerSource community is an excellent opportunity to acquire new skills
+and to grow personally. This is again where the role of the Trusted Committer is really
+important. Trusted Committers often become mentors for junior developers, and
+explicitly spend time during pull requests not only pointing out areas
+for improvement but also explaining in detail why something needs to be
+improved and how to do it. By doing so, Trusted Committers
```

Comment:
I liked the original, split up sentence better here since it appears more mentoring focused to me.